### PR TITLE
Beautify command box

### PR DESCRIPTION
**Pull request by wilzbach**
- Swap buttons and put the text left for a more natural feeling
- Add 5px spacing between the text and the button

image

image

See also PR #570 and #569

### CODE REVIEW

File: public/static/css/common.css [hunk 1]
```diff
@@ -103,10 +103,6 @@ body {
     border-color: #adadad;
 }
 
-button > i + span {
-	margin-left: 5px;
-}
```
Comment (wilzbach) [inline public/static/css/common.css]: While `common.css` has been altered before, initially it was though to contain global CSS definitions from dlang.org (might do a cleanup later).

Comment (ZombineDev) [inline public/static/css/common.css]: Sorry, I was too quick to merge.

File: views/editor.dt [hunk 2]
```diff
@@ -19,20 +19,20 @@ block content
 		div(style="padding-left: 0px; padding-right: 0px")
 			div#command-box.text-right
 				button.btn.btn-primary(ng-click="run()")
-					i.fa.fa-play(aria-hidden="true")
 					span Run
-				button.btn.btn-primary(ng-click="format()")
 					i.fa.fa-play(aria-hidden="true")
+				button.btn.btn-primary(ng-click="format()")
 					span Format
+					i.fa.fa-play(aria-hidden="true")
```
Comment (ZombineDev) [inline views/editor.dt]: I'd like to find a good alternative to reusing the play icon. I opened a separate issue for that: #572. (We should leave it for a separate PR, of course.)

File: public/static/css/common.css (hunk 1 above)
Comment (wilzbach) [inline public/static/css/common.css]: No worries.

File: views/editor.dt [hunk 3]
```diff
@@ -19,20 +19,20 @@ block content
 		div(style="padding-left: 0px; padding-right: 0px")
 			div#command-box.text-right
 				button.btn.btn-primary(ng-click="run()")
-					i.fa.fa-play(aria-hidden="true")
 					span Run
-				button.btn.btn-primary(ng-click="format()")
 					i.fa.fa-play(aria-hidden="true")
+				button.btn.btn-primary(ng-click="format()")
```
Comment (ZombineDev) [inline views/editor.dt]: I think the format button should use the `btn-default` style. It has the same importance as `Reset`, `Shorten`, etc., IMO.

**Comment by wilzbach** [comment]
> I'd like to find a good alternative to reusing the play icon. I opened a separate issue for that: #572. (We should leave it for a separate PR, of course.)

Yep I like initiative!
Let's use magic for the time being - it's definitely better than repeating the play button.

> I think the format button should use the `btn-default` style. It has the same importance as `Reset`, `Shorten`, etc., IMO.

Done:

image

image

**Comment by wilzbach** [comment]
> I think the format button should use the `btn-default` style. It has the same importance as `Reset`, `Shorten`, etc., IMO.

I think I stupidly just copied the "Run" button when we introduced "Format".

**Comment by wilzbach** [comment]
Overwriting Travis as it will fail due to https://issues.dlang.org/show_bug.cgi?id=17614 for dmd-beta anyhow & it's just a tested UI change.

**Comment by ZombineDev** [comment]
I was about to do that myself, but you were faster ;)
